### PR TITLE
Update Airtable

### PR DESCRIPTION
**Pull request by 40504**

### CODE REVIEW

File: entries/a/airtable.com.json [hunk 1]
```diff
@@ -6,7 +6,8 @@
       "call",
       "totp"
     ],
-    "documentation": "https://support.airtable.com/hc/en-us/articles/219409917",
+    "documentation": "https://support.airtable.com/docs/enabling-two-factor-authentication",
+    "recovery": "https://support.airtable.com/v1/docs/contacting-airtable-support",
```
Comment (hkamran80) [inline entries/a/airtable.com.json]: Maybe link to the FAQ section of the 2FA documentation instead, since it provides some context?

**Review by hkamran80 — CHANGES_REQUESTED** [review]

File: entries/a/airtable.com.json (hunk 1 above)
Comment (40504) [reply entries/a/airtable.com.json]: Could you suggest the URL? :)

Comment (hkamran80) [reply entries/a/airtable.com.json]:
```suggestion
    "recovery": "https://support.airtable.com/docs/enabling-two-factor-authentication#faqs",
```

**Review by hkamran80 — CHANGES_REQUESTED** [review]

**Review by Carlgo11 — APPROVED** [review]

**Comment by phallobst** [comment]
Do we generally want to have the `documentation` and `recovery` urls point to the same page, even if anchors are used?

**Comment by hkamran80** [comment]
I think we should have the URLs point to the same page if both pieces of information are present on that page. [WeTransfer's entry](https://github.com/2factorauth/twofactorauth/blob/master/entries/w/wetransfer.com.json) has the same URL with anchors because they have the information on one page.

**Review by hkamran80 — APPROVED** [review]

**Pull request merged**